mixins/localize-mixin.md

```diff
@@ -121,6 +121,65 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
+
+You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method.
+
+### Simple Markup
+
+#### Bold
+
+The `[b]bold[/b]` markup converts to the `<strong>` tag.
```

Comment:
Are we using the right token or tag names? I hate to second guess this, but this is probably worth reviewing before we commit one way or another. We seem to be using them interchangeably, where Mozilla seems to suggest we should not. Similar story for italic.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#usage_notes
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong

Reply:
And to be fair... their explanation doesn't make it much clearer to me. 😕 

>Don't they both do the same thing?"
...
>Not exactly. The `<strong>` element is for content that is of greater importance, while the `<b>` element is used to draw attention to text without indicating that it's more important.

Reply:
I intentionally chose to use `em` and `strong` as they have actual meaning to add to words within a sentence. I figured if you were trying to style something, you should probably be styling the entire thing, which can be done with CSS in your actual code.

Reply:
Coming back to this... `<strong>` vs. `<b>` and `<em>` vs. `<i>` is a [semantic vs. stylistic](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em) choice. I agree with Danny that styles can be applied outside of localization, which makes me want to _not_ support `<b>` or `<i>` at all -- only `<strong>` and `<em>`.